### DISCUSSION TITLE
Would it be possible to add Soul Break export?

### DISCUSSION THREAD

**Discussion by Dorian129** [HughJeffner/FFRK-LabMem#223] [Ideas]
I'm not sure how much trouble it would be, but since this has the certificatioms to do things with the proxy, would it be possible to incorporate a functionality to export your Soul Breaks to a text file or something?

**Comment by HughJeffner** [comment]
I could, but it's kinda out of scope for this project

This seems to have that function but I haven't tried it:

https://github.com/BaconCatBug/FFRK-Drop-Tracker

Looks like you use the 'export inventory' script then feed that data into a google doc
